aspnetcore/signalr/publish-to-azure-web-app.md

```diff
@@ -1,63 +1,80 @@
 
 ## Publish the app
 
-Visual Studio provides built-in tools for publishing to an Azure Web App. Visual Studio Code user can use [Azure CLI](/cli/azure) commands to publish apps to Azure. This article covers publishing using the tools in Visual Studio. To publish an app using Azure CLI, see [Publish an ASP.NET Core app to Azure with command line tools](/azure/app-service/app-service-web-get-started-dotnet).
+This article covers publishing using the tools in Visual Studio. Visual Studio Code users can use [Azure CLI](/cli/azure) commands to publish apps to Azure. For more information, see [Publish an ASP.NET Core app to Azure with command line tools](/azure/app-service/app-service-web-get-started-dotnet).
 
-Right-click on the project in **Solution Explorer** and select **Publish**. Confirm that **Create new** is checked in the **Pick a publish target** dialog, and select **Publish**.
+1. Right-click on the project in **Solution Explorer** and select **Publish**. Confirm that **App Service** and **Create new** are selected in the **Pick a publish target** dialog. Select **Create Profile** from the **Publish** button drop down.
 
-![Pick publish target](publish-to-azure-web-app/_static/pick-publish-target-dialog.png)
+   Enter the information described in the following table in the **Create App Service** dialog and select **Create**.
 
-Enter the following information in the **Create App Service** dialog and select **Create**.
+   | Item               | Description |
+   | ------------------ | ----------- |
+   | **Name**           | Unique name of the app. |
+   | **Subscription**   | Azure subscription that the app uses. |
+   | **Resource Group** | Group of related resources to which the app belongs. |
+   | **Hosting Plan**   | Pricing plan for the web app. |
 
-| Item | Description |
-| ---- | ----------- |
-| **App name** | A unique name of the app. |
-| **Subscription** | The Azure subscription that the app uses. |
-| **Resource Group** | The group of related resources to which the app belongs.  |
-| **Hosting Plan** | The pricing plan for the web app. |
+1. Select the **Azure SignalR Service** in the **Dependencies** > **Add** drop-down list:
 
-![Create app service](publish-to-azure-web-app/_static/create-app-service-dialog.png)
+   ![Dependencies area showing the selection of Azure SignalR Service in the Add drop-down list](publish-to-azure-web-app/_static/signalr-service-dependency.png)
+
+1. In the the **Azure SignalR Service** dialog, select **Create a new Azure SignalR Service instance**. Provide a **Name**, **Resource Group**, and **Location**. Return to the **Azure SignalR Service** dialog and select **Add**.
 
 Visual Studio completes the following tasks:
 
 * Creates a Publish Profile containing publish settings.
 * Creates or uses an existing *Azure Web App* with the provided details.
```

Comment:
The directions above say "create new", so it wouldn't be using an existing one here.